content/outcomes/application-development/cloud-native-development.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Cloud Native Development
+weight: 40
+layout: single
+---
+
+Cloud Native is at the heart of modern application development endeavors. Used to depict engineering practices and state of the art applications, the term "cloud-native" is omnipresent and often misunderstood. There is a good chance that anyone you ask will provide a different definition.
+
+![Cloud Native Development](/images/outcomes/app-dev-onboarding/cloud-native-development.jpg)
+
+## What you will learn
+
+In this article, you will learn to:
+
+- [Define one meaning of cloud native](#meaning)
+- [List three major outcomes of cloud native](#outcomes)
+- Explain the main cloud native concepts for ...
+	- [application architecture](#app-archi)
+	- [software life-cycle](#sw-life-cycle)
+	- [supporting infrastructure](#support-infra)
+	- [team culture](#culture)
+- [Identify the twelve factors of cloud native applications](#factors)
```

Comment:
The section links are a great idea. We should do this for all of the articles.

Reply:
I am glad that you like it. I just updated the balanced-team article accordingly.